### PR TITLE
updating test to trigger build

### PR DESCRIPTION
**Pull request by rshade**
the goal of this pull request is to make sure travis functions, and that lint works, as well as reviews. etc.

### CODE REVIEW

File: resources/client.rb [hunk 1]
```diff
@@ -1,7 +1,9 @@
 
 
 action :install do
-  package 'mysql'
+  mysql_client 'default' do
+    action :create
+  end
```
Comment (damacus) [inline resources/client.rb]: you should be able to do ` mysql_client 'default'` here if you don't intend to pass any values.

File: test/fixtures/cookbooks/test/recipes/default.rb [hunk 2]
```diff
@@ -6,13 +6,14 @@
 #
 # All rights reserved - Do Not Redistribute
 #
+include_recipe 'yum-mysql-community::mysql56' if node['platform_family'] == 'rhel'
```
Comment (damacus) [inline test/fixtures/cookbooks/test/recipes/default.rb]: amazon?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
 # mysql_database
 
-TODO: Enter the cookbook description here.
+lets steal from database cookbook so no duplication of effort :)
```
Comment (iennae) [inline README.md]: maybe put a description of what this cookbook is for here. This doesn't explain to a new person what the point of this cookbook is for. This looks more like a comment right now.

File: metadata.rb [hunk 4]
```diff
@@ -3,11 +3,16 @@
 issues_url       'https://github.com/sous-chefs/mysql_database/issues' if respond_to?(:issues_url)
 maintainer       'Sous Chefs'
 maintainer_email 'help@sous-chefs.org'
-chef_version     '>= 11' if respond_to?(:chef_version)
 license          'Apache 2 License'
-description      'Installs/Configures mysql_database'
+description      'Installs/Configures mysql_database client utilities and provides resources for management'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version          '0.1.0'
+chef_version '>= 12.5' if respond_to?(:chef_version)
```
Comment (iennae) [inline metadata.rb]: Recommend sticking at 12.7 to avoid any class_eval issues.

File: metadata.rb [hunk 5]
```diff
@@ -3,11 +3,16 @@
...
 
+%w( debian ubuntu centos redhat oracle amazon ).each do |os|
+  supports os
+end
 
+depends 'mysql'
 depends 'mysql2_chef_gem', '>= 2.0'
+depends 'compat_resource', '>= 12.16'
```
Comment (iennae) [inline metadata.rb]: does it need compat_resource?

File: .foodcritic [hunk 6]
```diff
@@ -0,0 +1 @@
+~FC016
```
Comment (tas50) [inline .foodcritic]: You shouldn't need to disable this with the new Foodcritic

File: metadata.rb [hunk 7]
```diff
@@ -3,11 +3,16 @@
...
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version          '0.1.0'
+chef_version '>= 12.5' if respond_to?(:chef_version)
 
+%w( debian ubuntu centos redhat oracle amazon ).each do |os|
+  supports os
+end
 
+depends 'mysql'
 depends 'mysql2_chef_gem', '>= 2.0'
```
Comment (tas50) [inline metadata.rb]: Long term we should wrap this gem install into this cookbook. This is a needless dep

Comment (rshade) [inline metadata.rb]: https://github.com/sous-chefs/mysql_database/issues/10

File: metadata.rb (hunk 5 above)
Comment (rshade) [inline metadata.rb]: if we want to support chef12 we need compat_resource I believe.

File: test/fixtures/cookbooks/test/recipes/default.rb (hunk 2 above)
Comment (rshade) [inline test/fixtures/cookbooks/test/recipes/default.rb]: Amazon works without the change

Comment (damacus) [inline test/fixtures/cookbooks/test/recipes/default.rb]: Not anymore 
https://github.com/chef/chef/issues/4665
https://github.com/chef/ohai/blob/master/lib/ohai/plugins/linux/platform.rb#L112

Comment (damacus) [inline test/fixtures/cookbooks/test/recipes/default.rb]: Unless it doesn't need that cookbook? *

Comment (rshade) [inline test/fixtures/cookbooks/test/recipes/default.rb]: it doesn't only RHEL stripped out the mysql stuff in favor of mariadb

Comment (damacus) [inline test/fixtures/cookbooks/test/recipes/default.rb]: Awesome! Thanks

File: metadata.rb (hunk 5 above)
Comment (iennae) [inline metadata.rb]: it looks like this is getting pulled in by another cookbook. I don't think this cookbook does anything directly with compat_resource at this time which is why I brought up the question about needing it.

**Pull request merged**